collection-scripts/gather_serving

```diff
@@ -8,6 +8,8 @@ resources+=("authconfigs" "authorinos" "authpolicies.kuadrant.io")
 resources+=("accounts.nim.opendatahub.io")
 # Dependent resources llm-d
 resources+=("llminferenceserviceconfigs" "llminferenceservices" "leaderworkersetoperators" "leaderworkersets" "gateways.gateway.networking.k8s.io" "inferencepools")
+# Dependent resources MaaS
+resources+=("ratelimitpolicies.kuadrant.io" "kuadrants.kuadrant.io" "tokenratelimitpolicies.kuadrant.io")
```

Comment:
I am not sure we need `kuadrant.io` suffix, but we have it for authpolicies so maybe better to keep it consistent and complete (but that would mean `"authconfigs" "authorinos"` need a suffix too?). 

We might want to add:


```suggestion
resources+=("ratelimitpolicies.kuadrant.io" "kuadrants.kuadrant.io" "tokenratelimitpolicies.kuadrant.io" "telemetrypolicies.extensions.kuadrant.io")
```